code/datums/diseases/advance/symptoms/skin.dm

```diff
@@ -333,3 +333,206 @@ BONUS
 		icon_state = "fleshegg-sneaky"
 	if(LAZYLEN(diseases))
 		AddComponent(/datum/component/infective, diseases)
+
+/*
+//////////////////////////////////////
+Spiked Skin
+	Noticeable.
+	Raises resistance
+	Decreases stage speed
+	No transmission bonus
+
+Thresholds
+  transmission 6 Can give minor armour
+  resistance 6 Does more damage
+
+//////////////////////////////////////
+*/
+/datum/symptom/spiked
+	name = "Cornu Cutaneum"
+	desc = "The virus causes the host to unpredictably grow and shed sharp spines, damaging those near them."
+	stealth = -3
+	resistance = 3
+	stage_speed = -3
+	transmission = 0
+	level = 0
+	symptom_delay_min = 1
+	symptom_delay_max = 1
+	severity = 1
+	base_message_chance = 5
+	prefixes = list("Thorny ", "Horned ")
+	bodies = list("Horn", "Spiked")
+	var/armor = 0
+	var/done = FALSE
+	threshold_desc = "<b>Transmission 6:</b> Spikes deal more damage.<br>\
+					  <b>Resistance 6:</b> Hard spines give the host armor, scaling with resistance."
+
+/datum/symptom/spiked/severityset(datum/disease/advance/A)
+	. = ..()
+	if(A.resistance >= 6)
+		severity -= 1
+
+/datum/symptom/spiked/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.resistance >= 6) //armor. capped at 20, but scaling with resistance, so if you want to max out spiked skin armor, you'll have to make several sacrifices
+		armor = min(20, A.resistance)
+	if(A.transmission >= 6) //higher damage
+		power = 1.4  //the typical +100% is waaaay too strong here when the symptom is stacked. +40% is sufficient
+
+/datum/symptom/spiked/Activate(var/datum/disease/advance/A)
+	var/mob/living/carbon/human/H = A.affected_mob
```

Comment:
Unchecked human typecast

Reply:
out of scope, but ill fix it. this doesnt need to only work for humans